### PR TITLE
history 5.4.5

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

entry for the coming release

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -5,6 +5,21 @@
 OMERO version history
 =====================
 
+5.4.5 (March 2018)
+--------------------
+
+This is a bug fix release reactivating the thumbnail cache
```
Comment (hflynn) [inline history.txt]: bug-fix

File: history.txt [hunk 2]
```diff
@@ -5,6 +5,21 @@
...
+inadvertently disabled in 5.4.4 while fixing a pyramid issue
```
Comment (hflynn) [inline history.txt]: Missing full stop at the end of the sentence.

File: history.txt [hunk 3]
```diff
@@ -5,6 +5,21 @@
...
+
+Improvements include:
+
+- reactivated thumbnail caching
+- improved removepyramids help
+- fixed display of thumbnail when searching for images by id
```
Comment (hflynn) [inline history.txt]: thumbnails
ID

**Review by hflynn — COMMENTED** [review]
Just a few minor nitpicks

**Comment by hflynn** [comment]
Only other thing is, does the list formatting match previous entries? Looks like we've got a doublespace between the dash and the text for the next entry down.

**Comment by jburel** [comment]
@hflynn done

**Review by hflynn — APPROVED** [review]
Thanks, all looks good to me now

File: history.txt [hunk 4]
```diff
@@ -5,6 +5,21 @@
...
+This is a bug-fix release reactivating the thumbnail cache
+inadvertently disabled in 5.4.4 while fixing a pyramid issue.
+
+Improvements include:
+
+-  reactivated thumbnail caching
+-  improved removepyramids help
+-  fixed display of thumbnails when searching for images by ID
+-  increased OMERO.web log size
+-  fixed CLI list subcommand
```
Comment (hflynn) [inline history.txt]: @mtbc asked me to change this to "fixed CLI config list subcommand" on the website PR so you should probably update this match. Thanks

**Comment by jburel** [comment]
@hflynn done

**Comment by hflynn** [comment]
👍

**Pull request merged**